packages/webapp/pages/posts/[id].tsx

```diff
@@ -243,23 +244,30 @@ const PostPage = ({ id, postData }: Props): ReactElement => {
     <>
       <div className="flex relative flex-row flex-wrap justify-center pb-20 laptop:pb-0 w-full max-w-full">
         <PageContainer
-          className={classNames(
-            'pt-6 laptop:pb-6 laptop:self-stretch',
-            pageBorders,
-          )}
+          className={classNames('laptop:pb-6 laptop:self-stretch', pageBorders)}
         >
           <Head>
             <link rel="preload" as="image" href={postById?.post.image} />
           </Head>
           <NextSeo {...seo} />
+          <a
+            {...postLinkProps}
+            className="block overflow-hidden mb-10 rounded-b-2xl cursor-pointer"
+          >
+            <LazyImage
+              imgSrc={postById?.post.image}
+              imgAlt="Post cover image"
+              ratio="21%"
+              eager
+            />
+          </a>
           <PostHeader post={postById.post} />
-
           <a {...postLinkProps} className="cursor-pointer">
-            <h1 className="my-2 font-bold typo-title2">
+            <h1 className="my-2 font-bold typo-large-title">
               {postById?.post.title}
             </h1>
           </a>
-
+          <PostTLDRSummary post={postById.post} />
```

Comment:
I think we should dynamically render this only if the `post.summary` is actually available - this would display the TLDR even summary should not be available.